slide-deck.js

```diff
@@ -63,6 +63,41 @@ class slideDeck extends HTMLElement {
   static #adoptShadowStyles = (node) => {
     const shadowStyle = new CSSStyleSheet();
     shadowStyle.replaceSync(`
+      [part=control-panel] {
+        --sd-panel-gap: clamp(8px, 0.25em + 1vw, 24px);
+        min-width: min(50ch, 100%);
+        padding: 0;
+      }
+
+      [part=panel-header] {
+        align-items: center;
+        border-block-end: thin solid gray;
+        display: grid;
+        gap: var(--sd-panel-gap);
+        grid-template-columns: 1fr auto;
+        padding: var(--sd-panel-gap);
+      }
+
+      [part=controls] {
+        padding: var(--sd-panel-gap);
+      }
+
+      hr {
+        border-block-start: thin dotted gray;
+        border-block-end: none;
+        margin-block: 1lh;
+      }
+
+      [part~=button] {
+        border: medium solid transparent;
+        font: inherit;
+        padding-inline: var(--sd-panel-gap);
+
+        &[aria-pressed=true] {
+          border-color: currentColor;
+        }
+      }
```

Comment:
@mirisuzanne is this supposed to work on keyboard navigation? It looks like pressing Keyboard Navigation button in Script view control panel, I'm not sure it does anything but also, I get no feedback that it is toggle-able?